framework/src/modules/http_api/controllers/node.js

```diff
@@ -326,4 +317,27 @@ async function _getNetworkHeight() {
 	return parseInt(networkHeight);
 }
 
+/**
+ * Parse transaction instance to raw data
+ *
+ * @returns Object
+ * @private
+ */
+function _parseTransaction(transaction) {
```

Comment:
maybe improve name is it parses and returns a new object?

Reply:
Any suggestion?

Reply:
`normalizeTransaction` maybe?